x-pack/plugins/infra/server/routes/log_entries/summary.ts

```diff
@@ -47,6 +48,8 @@ export const initLogEntriesSummaryRoute = ({ framework, logEntries }: InfraBacke
           parseFilterQuery(query)
         );
 
+        UsageCollector.countLogs();
```

Comment:
Should this be in its own try/catch so we don't bounce to an error message just because the usage collector fails?

Reply:
Good point, but if the data collection might fail I think it should be handled inside the telemetry call itself and not in the application code. I think telemetry should behave like a black box for app code